Project02Eclipse/src/com/hemebiotech/analytics/WriteSymptomDataResult.java

```diff
@@ -0,0 +1,36 @@
+package com.hemebiotech.analytics;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
+public class WriteSymptomDataResult implements ISymptomResultWriter{
+    private final FileWriter writer = new FileWriter("result.out");
+
+    /**
+     *
+     * @throws IOException
+     */
+    public WriteSymptomDataResult() throws IOException {}
+
+    /**
+     *
+     * @param listSymptomsClean
+     * @throws IOException
+     */
+    public void WriteSymptoms(Hashtable<String, Integer> listSymptomsClean ) throws IOException {
+        Set<String> keys = listSymptomsClean.keySet();
+        Iterator<String> itr = keys.iterator();
```

Comment:
We should program by intention, please avoid using short name. This one should be renamed `iterator`.